### ISSUE TITLE
Unable to compile mod_auth_vas on Solaris 11.3

### ISSUE THREAD

**Issue by pru-qmir** [OneIdentity/mod_auth_vas#6]
We recently upgraded vintela to QAS 4.2.2.25319. It broke MAV 3.6.8.4 for HTTPD 2.2.

Unable to compile and install on Solaris 11.3 -
- MAV 3.6.8.5 for HTTPD 2.2
- MAV4 4.0.2.4 for HTTPD 2.4

Source code configure failing with below errors -

- MAV 3.6.8.5 for HTTPD 2.2
configure: error: in `/var/tmp/mod_auth_vas-3.6.8.5':
configure: error: VAS 3 SDK required

- MAV4 4.0.2.4 for HTTPD 2.4
configure: error: in `/var/tmp/mod_auth_vas4-4.0.2.4':
configure: error: VAS 3 SDK required

Need help to complie or get a pre-compiled latest MAV and MAV4 packages for Solaris 11.3 running QAS 4.2.2.25319

**Comment by pru-qmir** [comment]
[mod_auth_vas-3.6.8.5-config.log](https://github.com/OneIdentity/mod_auth_vas/files/5617170/mod_auth_vas-3.6.8.5-config.log)
[mod_auth_vas4-4.0.2.4-config.log](https://github.com/OneIdentity/mod_auth_vas/files/5617171/mod_auth_vas4-4.0.2.4-config.log)

One Identiity SR Number:4782095

**Comment by manner82** [comment]
Started checking this today, sorry for the delay.

Seems it fails to compile against VAS library, therefor the error. And it fails to do so, because some options get into CFLAGS,
which gcc does not support: (these are from your logs)
gcc: error: cg: No such file or directory
gcc: error: unrecognized command line option '-Qoption'
gcc: error: unrecognized command line option '-W2,-xwrap_int'
gcc: error: unrecognized command line option '-mt'; did you mean '-t'?

In my experiment, I get only the error about the '-mt' argument. That one (bad CFLAG) seems to come from the invocation of the command:
apxs -q CFLAGS

So the source of the problem might be that apache was compiled with a different compiler. In case you found out something in the meantime, don't hesitate to share :)

**Comment by manner82** [comment]
I think we can approach the compiler flag differences by skipping the flags the current compiler does not understand (and warning the user). So with a chance it can still work out of box, otherwise (s)he still can provide them as CFLAGS argument to the configure script.
I have added this logic at my clone: https://github.com/manner82/mod_auth_vas/tree/solaris_autotools_fixes

I also bumped in to some other issues:
 - "echo -n" not working (fix is on my branch)
 - probably x86 specific: compiling against 64 bit apache adds -m64 flag: this causes some problems, eg linking with 32 bit vas libraries (I needed to manually adjust the path where they get searched).
 - needed to add apr-util include path because it was different from apr include path on my system. (I have added autodetection for that.)
 - missing vas_symbols/gssapi_sym.h: this seems to not present in vasdev 4.2.2.25319, but present in 5.0.0 so might got a fix in between those two versions
